tests/test_rooms/test_query_rooms.py

```diff
@@ -34,13 +35,24 @@ def test_paginate_room_query(self):
 
     @patch("api.room.schema_query.get_google_api_calendar_list", spec=True,
            return_value=get_calendar_list_mock_data())
-    def test_query_remote_rooms(self, mock_get_json):
+    @patch.dict(os.environ, {"APP_SETTINGS": "production"})
+    def test_query_actual_remote_rooms(self, mock_get_json):
         CommonTestCases.admin_token_assert_in(
             self,
             all_remote_rooms_query,
             "calendar.google.com"
         )
 
+    @patch("api.room.schema_query.get_google_api_calendar_list", spec=True,
```

Comment:
Why do you need this patch here? Also, why are you asserting that an empty string is in the response? Of course, that will be true.

Reply:
I am using the patch to set the APP_SETTINGS value to either production or development.

Reply:
I am talking about the one on line 46.